tools/uglifyjs-pathfix

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+# Script for docker: The uglifyjs installed by debian uses nodejs from $PATH, which may be a nodejs that comes with emsdk, which doesn't have the necessary modules to run uglifyjs. So strip emsdk from path before running uglifyjs.
+
+set -euo pipefail
+self="$(realpath "$0")"
+export PATH="$(echo -n "$PATH" | tr : \\0 | grep -vz emsdk | grep -vz "$(dirname "$self")" | tr \\0 :)"
```

Comment:
Could we set this locally instead of changing the path globally? https://linux.die.net/abs-guide/localvar.html

This would be a bit difficult to maintain. Is installing uglifyjs for the node shipped with emsdk an option? Let's see if this is still an issue with https://github.com/iodide-project/pyodide/pull/660

Reply:
I'm not sure what you mean with localvar. Even with export, the changed `PATH` is still only visible to this bash and the uglifyjs process, right? If uglifyjs doesn't invoke anything else, this should be fine.
I can of course remove the export, after which the `PATH` change would only be visible to that one `exec` in that script. But I'm not sure that's correct/sufficient.

Alternatively, one could just `exec /usr/bin/node "$(which uglifyjs)"` and delete the rest of the script. That hardcodes node's path, but I doubt that's gonna change (again…), and it's easy to understand and fix.

Installing uglifyjs to the emsdk node is probably not so easy, since `/src/emsdk/emsdk/node/8.9.1_64bit/bin/npm` chokes on itself being too outdated.

Reply:
OK, I wasn't sure if the export was global or not.

> Alternatively, one could just exec /usr/bin/node "$(which uglifyjs)" and delete the rest of the script. 

Yeah, let's do that. It would be easier to understand and fix if needed. 


https://github.com/iodide-project/pyodide/pull/660 with debian buster has some issues, so let's merge this PR first.

Reply:
(got rid of of the external script, as exec is simple enough. haven't tested it yet.)